### PR TITLE
3516 Enable code splitting

### PR DESCRIPTION
**Pull request by ericsorenson**
enable code splitting, with pdfStyles as an example

### CODE REVIEW

**Review by matthopson — APPROVED** [review]

File: web-client/src/applicationContext.js [hunk 1]
```diff
@@ -468,11 +467,12 @@ const applicationContext = {
...
   getFileReader: () => FileReader,
   getHttpClient: () => axios,
-  getPdfJs: () => {
+  getPdfJs: async () => {
     pdfjsLib.GlobalWorkerOptions.workerSrc = '/pdf.worker.min.js';
     return pdfjsLib;
   },
-  getPdfStyles: () => {
+  getPdfStyles: async () => {
+    const pdfStyles = await import('../../shared/src/tools/pdfStyles.js');
```
Comment (kkoskelin) [inline web-client/src/applicationContext.js:475]: Tell me more about this.

File: web-client/src/applicationContext.js [hunk 2]
```diff
@@ -468,11 +467,12 @@ const applicationContext = {
   },
   getFileReader: () => FileReader,
   getHttpClient: () => axios,
-  getPdfJs: () => {
+  getPdfJs: async () => {
     pdfjsLib.GlobalWorkerOptions.workerSrc = '/pdf.worker.min.js';
     return pdfjsLib;
   },
-  getPdfStyles: () => {
+  getPdfStyles: async () => {
```
Comment (kkoskelin) [inline web-client/src/applicationContext.js:474]: Now that this is `async`, you may be needing an `await` in at least one other place (according to my search results)

**Review by kkoskelin — CHANGES_REQUESTED** [review]

File: web-client/src/applicationContext.js (hunk 2 above)
Comment (ericsorenson) [reply web-client/src/applicationContext.js:474]: Updated!

File: web-client/src/applicationContext.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/applicationContext.js:475]: Lazy loading of code keeps the bundle size small.

**Review by kkoskelin — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**